web_development_101/javascript_basics/Untitled-1

```diff
@@ -0,0 +1,16 @@
+take a number n
```

Comment:
You've added a file that shouldn't go into the curriculum repo here, can you remove this please?